.circleci/config.yml

```diff
@@ -9,16 +9,11 @@ defaults: &defaults
     - image: cimg/node:14.19
   working_directory: /mnt/ramdisk/repo
 
-# mac: &mac
-#   macos:
-#     xcode: "13.2.1"
-#   working_directory: ~/repo
-
```

Comment:
Removed comments. They are in the git history, so we can get them back. Otherwise they create a lot of noise